python/cugraph/cugraph/tests/layout/test_force_atlas2.py

```diff
@@ -27,15 +27,13 @@
     dining_prefs,
 )
 
-# Temporarily suppress warnings till networkX fixes deprecation warnings
-# (Using or importing the ABCs from 'collections' instead of from
-# 'collections.abc' is deprecated, and in 3.8 it will stop working) for
-# python 3.7.  Also, these import fa2 and import networkx need to be
-# relocated in the third-party group once this gets fixed.
-
+# FIXME Removed the multi column positional due to it being non-deterministic
+# need to replace this coverage. Issue 3890 in cuGraph repo was created.
```

Comment:
I can't find issue 3890 in the cugraph repo. Are you sure this is the right number?

Reply:
I'm assuming you meant [issue 3900](https://github.com/rapidsai/cugraph/issues/3900) ?